dotcom-rendering/src/server/prod-server.ts

```diff
@@ -75,6 +78,7 @@ export const prodServer = (): void => {
 		handleAllEditorialNewslettersPage,
 	);
 	app.post('/AppsArticle', logRenderTime, handleAppsArticle);
+	app.post('/AppsInteractive', logRenderTime, handleAppsInteractive);
```

Comment:
Should the `prod-server` also have a GET route like here https://github.com/guardian/dotcom-rendering/pull/7228/files?

Reply:
ah yes, thanks for spotting, I'll add that now